source/community/reactnative/src/components/inputs/input.js

```diff
@@ -233,12 +233,12 @@ export class Input extends React.Component {
 							value={this.props.value}
 						/>
 						<TouchableOpacity onPress={() => { this.onClickHandle(onExecuteAction, 'inline-action') }}>
-							{Utils.isNullOrEmpty(inlineAction.iconUrl) ?
+							{Utils.isNullOrEmpty(inlineAction.url) ?
```

Comment:
It is iconUrl for action elements. Please refer to the schema - https://adaptivecards.io/explorer/Action.OpenUrl.html